packages/app/src/types/verifiableCredentials/index.ts

```diff
@@ -10,17 +10,17 @@ export interface ICryptkeeperVerifiableCredential {
   metadata: IVerifiableCredentialMetadata;
 }
 
-export interface IAddVerifiableCredentialArgs {
+export interface IAddVCArgs {
   serializedVerifiableCredential: string;
   verifiableCredentialName: string;
```

Comment:
```suggestion
  serializedVC: string;
  vcName: string;
```

Reply:
It can be just name and serialized. Interface has the main name and no need to specify the same in object params.